### DISCUSSION TITLE
Hide the label

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#4865] [Help] [answered]
Hello,

didn't find anything in the docs about it. I plan to use the forms package and sometimes I want to hide the label. Is this possible? If so, how?

Thank you in advance.

**Comment by danharrin** [comment]
`disableLabel()`

**Reply by danharrin** [reply]
The label still exists for screen readers

**Reply by shanerbaner82** [reply]
It looks like this method was deprecated in v3? Is there an alternate?

**Reply by danharrin** [reply]
All deprecation messages contain a replacement, this one is hiddenLabel() I think for consistency

**Reply by Wulfheart** [reply, marked as answer]
v2: `disableLabel()`
v3: `hiddenLabel()`

**Reply by kokoshneta** [reply]
This doesn’t seem to have any effect on `MorphToSelect` components – any way to hide the label and the border that’s generated around those?

**Reply by danharrin** [reply]
Hmm, try `->view('filament-forms::components.group')`

**Reply by kokoshneta** [reply]
Perfect, that worked – very clever workaround! (I’m still on v2, so just `forms::components.group` for me, but other than that.)

**Reply by danharrin** [reply]
yeah, thats a weird one

**Reply by micraux** [reply]
Pushing forward with @danharrin's clever workaround, if you want to render the `MorphToSelect` fields side by side, just add:
```php
->view('filament-forms::components.group')
->extraAttributes(['class' => '*:flex *:gap-2'])
```

**Reply by danharrin** [reply]
Can't you do it with `columns(2)`?

**Reply by micraux** [reply]
Yes, it works too ;-)
